DevelopGitHub/GitHubExtension/GitHubExtension.Infrastructure.ActionFilters/InternalActivitiesFilters/InternalActivityFilter.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System.Web.Http.Filters;
+
+namespace GitHubExtension.Infrastructure.ActionFilters.InternalActivitiesFilters
+{
+    public abstract class InternalActivityFilter : ActionFilterAttribute { }
```

Comment:
will change to  interface wich marked my attributes
public interface IInternalActivityFilter { }